src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/ParseRequestServiceTest.java

```diff
@@ -146,7 +162,7 @@ void prepareDeleteRequest_AssignerIdHeader() {
         when(correlationInterceptorUtilMock.preHandle(
             any(HttpServletRequest.class))).thenReturn("21334a2b-79ce-44eb-9168-2d49a744be9d");
 
-        Request builtReq = TestDataBuilder.buildRequest(CREATED, false);
+        Request builtReq = TestDataBuilder.buildRequest(Status.CREATED, false);
```

Comment:
you can import the static method of CREATED instead of writing Status.CREATED.

Reply:
Changed